src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider adding a fallback font-family in case the main one doesn't load. For example, 'Roboto, Arial, sans-serif' could be changed to 'Roboto, "Helvetica Neue", Arial, sans-serif'.